src/test/java/org/primeframework/mvc/GlobalTest.java

```diff
@@ -2308,6 +2316,27 @@ public void post_unsupported_multipleParameters() throws Exception {
         "You are attempting to map a form field that contains multiple parameters to a property on the action class that is of type Enum. This isn't allowed. Action class [org.example.action.ParameterHandlerAction] Request URI [/parameter-handler] Parameter name [enumValue]"));
   }
 
+  @Test
+  public void post_multipart_parameter_mix() throws IOException {
```

Comment:
Can we follow the existing pattern for these tests? 

Feel free to still leave your comments, but ideally they would be using the fluent builder. 

For example:
```

  @Test
  public void post_multipart_parameter_mix() throws Exception {
    // arrange
    test.createFile("Hello World")
        .simulate(() -> simulator

            // act
            .test("/user/full-form")
            .withParameter("roleIds", 21)
            .withParameter("roleIds", 22)
            .withURLParameter("ages", 42)
            .withFile("image", test.tempFile, "text/plain")
            .post()

            // assert
            .assertStatusCode(200)
            .assertBodyIsEmpty());


    assertEquals(FullFormAction.getRoleIdsFromLastInvocation().size(), 2);
    assertEquals(FullFormAction.getAgesFromLastInvocation().size(), 1);
    var fileContents = Files.readString((FullFormAction.getImageFromLastInvocation().getFile()));
    assertEquals(fileContents, "Hello World");
  }
  ```

Reply:
Done